packages/react/src/patterns/sub-patterns/TableOfContents/TableofContentsDesktop.js

```diff
@@ -23,7 +23,7 @@ const { prefix } = settings;
  * @param {*} props.updateState function to update parent state.
  * @returns {*} JSX Object
  */
-const TOCDesktopMenu = ({ menuItems, selectedId, updateState }) => {
+const TOCDesktop = ({ menuItems, selectedId, updateState }) => {
```

Comment:
Why was this filename renamed to `TableOfContentsDesktop`, but `TOCMobile` remained the same?

Reply:
Build is failing because of the prettier if i change the file name to TOCDesktop ... Please suggest a better file name so that i can go and update ......
<img width="1380" alt="Screen Shot 2020-01-31 at 1 14 51 PM" src="https://user-images.githubusercontent.com/12287070/73697483-a84a9200-46ac-11ea-8133-3e1dd4ca2c01.png">

Reply:
Have you found all references for TOCDesktop or TableOfContentsDesktop? There might be an import that's out of sync.

Reply:
Also, the name of the file should be `TOCDesktop.js` if that is the file that `TableOfContents.js` is pointing to.